powerline/segments/common/env.py

```diff
@@ -166,6 +168,8 @@ def user(pl, hide_user=None):
 		return None
 	if username == hide_user:
 		return None
+	if hide_domain == True:
+		username = username[0:username.find('@')]
```

Comment:
Also I do not see them deprecated.

Note: I meant `str.find` (`str` is a type), *not* `string.find` (`string` is a module containing lots of deprecated stuff).